pkg/importer/dataStream.go

```diff
@@ -266,6 +270,86 @@ func (d *DataStream) http() (io.ReadCloser, error) {
 	return countingReader, nil
 }
 
+const (
+	//ContainerDiskImageDir - Expected disk image location in contianer image as described in
+	//https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md
+	ContainerDiskImageDir = "disk"
+	//TempContainerDiskDir - Temporary location for pulled containerImage
+	TempContainerDiskDir = "/tmp"
```

Comment:
So, there is no way to use the PVC for temp space?  fakeroot did not work?

Reply:
Now storing on PVC in /data/tmp directory which is deleted at the end of the flow